gbpservice/neutron/tests/unit/nfp/configurator/agents/test_firewall.py

```diff
@@ -170,6 +175,8 @@ def _test_handle_event(self, rule_list_info=True):
                                             {'firewall_rule_list': True})
 
             agent.handle_event(self.ev)
+            if 'service_info' in self.fo.context:
```

Comment:
This change is needed as Ahmed put a temporary fix in firewall to clear service_info dict inside the context. This was added as there is an issue with large data on the socket. Once we fix it, this can be removed.